source/opt/def_use_manager.h

```diff
@@ -53,13 +54,41 @@ inline bool operator<(const Use& lhs, const Use& rhs) {
   return lhs.operand_index < rhs.operand_index;
 }
 
-using UseList = std::list<Use>;
+using UserEntry = std::pair<ir::Instruction*, ir::Instruction*>;
+
+struct less_than_id_ptr {
```

Comment:
Suggest renaming this to UserEntryLess

Reply:
Should say what this is trying to do.  It's a weak less-than ordering (or whatever), as required for the comparator for a std::set.

Reply:
Will comment.